src/web/components/elements/TextBlockComponent.tsx

```diff
@@ -137,6 +137,7 @@ export const TextBlockComponent: React.FC<Props> = ({
 	const paraStyles = css`
 		margin-bottom: 16px;
 		${body.medium()};
+        line-break: anywhere;
```

Comment:
Nitpick: This might be a syptom of us changing the style linter recently but that tabbing seems off.

Reply:
Ah gawd, does that mean I can't edit on Github anymore

Reply:
`line-break: anywhere` Oh! That may be better that `break-word` we applied somewhere recently (ie. will break URLs at hyphens and punctuation instead of middle of the word?). Asking, coz I don’t understand [the definition](https://developer.mozilla.org/en-US/docs/Web/CSS/line-break).

Reply:
@paperboyo here's some examples: https://codepen.io/gtrufitt/pen/NWRJRQo

Reply:
`line-break: anywhere` does seem to fair up better in those examples?